extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js

```diff
@@ -66,6 +66,8 @@ RdfSkeletonListDialog.prototype._renderPreview = function(schema) {
     this._nodeUIs.push(element);
   }
   $(".rdf-schema-list-table-layout").find('*').unbind('click');
+  $(".rdf-schema-list-table-layout").find('a:contains(add property)').remove();
+  $(".rdf-schema-list-table-layout").find('a:contains("add rdf:type")').remove();
```

Comment:
Non-consistent api usage: the first property is quoted, the second one isn't. Probably to escape the colon character? Might be better to be consistent, and use quotes for both. I'd propose to move the double quotes to the outside, and use the single quotes to quote the selector literal.

Reply:
will do